app/test/ui/widgets/atr_preview_test.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:app/ui/widgets/atr_preview.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import '../../test_animal_transport_record.dart';
+
+void main() {
+  group('Animal Transport Record Preview', () {
+    testWidgets('Previews right information', (WidgetTester tester) async {
+      final testDate = DateTime.now();
+      final testCompany = "Test Company";
+      final testSpecies = "Cows";
+
+      // Must wrap widget in MaterialApp if not "home" widget under test
+      await tester.pumpWidget(new MaterialApp(
```

Comment:
Related to [this discussion](https://github.com/flutter/flutter/issues/11859). It's not very obvious, and it doesn't look like they're gonna make it more obvious either.